distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestFederation.java

```diff
@@ -296,8 +306,83 @@ public void testOpensearchToCswSourceToCswEndpointQuerywithMetacardXml() throws
         String queryUrl = OPENSEARCH_PATH + "?q=" + DEFAULT_KEYWORD + "&format=xml&src="
                 + CSW_SOURCE_WITH_METACARD_XML_ID;
 
-        when().get(queryUrl).then().log().all().assertThat().body(containsString(RECORD_TITLE_1),
-                containsString(RECORD_TITLE_2));
+        when().get(queryUrl).then().log().all().assertThat()
+                .body(containsString(RECORD_TITLE_1), containsString(RECORD_TITLE_2));
+    }
+
+    @Test
+    public void testListAllSourceInfo() {
+        try {
+            setupConnectedSources();
+        } catch (IOException e) {
+            LOGGER.error("Couldn't create connected sources: {}", e.getMessage());
+        }
+
+        String queryUrl = ADMIN_SOURCE_PATH + ADMIN_ALL_SOURCE_INFO_PATH;
+        given().auth().basic("admin", "admin").when().get(queryUrl).then().log().all().assertThat()
+                .body(containsString("\"fpid\":\"OpenSearchSource\""),
+                        containsString("\"fpid\":\"Csw_Federated_Source\""),
+                        containsString("\"fpid\":\"Csw_Connected_Source\""));
+    }
+
+    @Test
+    public void testFederatedSourceStatus() {
+        String sourcesURL = ADMIN_SOURCE_PATH + ADMIN_ALL_SOURCE_INFO_PATH;
+        String statusURL = ADMIN_SOURCE_PATH + ADMIN_SOURCE_STATUS_PATH + "/";
+
+        // Find and test OpenSearch Federated Source
+        String openSearchPid = "";
+        String json = given().auth().basic("admin", "admin").when().get(sourcesURL).asString();
+        List<Map<String, Object>> sources = from(json).get("value");
+        for (Map<String, Object> source : sources) {
+            if (source.containsKey("configurations") && source.get("id").equals("OpenSearchSource")) {
+                ArrayList<Map<String, Object>> configs = (ArrayList<Map<String, Object>>) source
+                        .get("configurations");
+                openSearchPid = ((String) configs.get(0).get("id"));
+                break;
+            }
+        }
+
+        given().auth().basic("admin", "admin").when().get(statusURL + openSearchPid).then()
+                .log().all().assertThat().body(containsString("\"value\":true"));
+    }
+
+    // TODO: Connected csw/wfs sources are broken. Ticket: DDF-1366
+
```

Comment:
Include the Test annotation as well as the Ignore annotation.  This just makes it easy for us to look at the unit test reports to see if we are ignoring any tests.